pkg/proto/configuration/filesystem/virtual/virtual.proto

```diff
@@ -133,6 +133,9 @@ message FUSEMountConfiguration {
   //
   // Recommended value: unset
   map<string, string> linux_backing_dev_info_tunables = 9;
+
+  // List of options to pass directly to fusermount.
+  repeated string options = 10;
```

Comment:
Could we describe what the semantics of this option are if direct_mount is used?